keyboards/magic_force/mf34/readme.md

```diff
@@ -1,3 +1,6 @@
+# DEPENDING ON WHERE OR WHEN YOU PURCHASED THIS KEYBOARD, YOU MAY HAVE A DIFFERENT MCU AND BOOTLOADER THAN WHAT THIS FIRMWARE IS LOOKING FOR
```

Comment:
This should not be at the top of the file, also the suggestions for `Geehy Programmer ` are redundant as you can still use dfu-util if the right `DFU_ARGS` are used.